src/igvfd/audit/measurement_set.py

```diff
@@ -88,7 +88,7 @@ def audit_unspecified_protocol(value, system):
 @audit_checker('MeasurementSet', frame='object')
 def audit_inconsistent_readout(value, system):
     '''
-        audit_detail: CRISPR-based and MPRA assays are required to specify a readout that is different from the assay term of the measurement set, other assays should not include readout specification.
+        audit_detail: CRISPR-based and MPRA measurement sets are required to specify a readout (not compliant) that is different from the assay term (error), other assays are not expected to include a readout specification (error).
```

Comment:
CRISPR-based and MPRA measurement sets are required to specify a readout (NOT COMPLIANT) that is different from the assay term (ERROR).  Other assays are not expected to include a readout specification (ERROR).

Reply:
Ok.